File: Php/app/Auth/AuthLoginModel.php
<?php
namespace app\Auth{

	use app\Urls\ForgotPasswordUrlModel;
	use core\PageModel;
	use app\Urls\UserAddUrlModel;

	class AuthLoginModel extends PageModel {

		public $returnUrl;
		public $addUserUrl;
		public $forgotPasswordUrl;
		public $loginName;

		public function __construct($returnUrl = null, $loginName){
			parent::__construct();
			$this->returnUrl = $returnUrl;
			if($this->returnUrl == null){
				$this->returnUrl = '/';
			}
			$this->addUserUrl = new UserAddUrlModel();
			$this->forgotPasswordUrl = new ForgotPasswordUrlModel();
			if($loginName != null){
				$this->loginName = $loginName;
			}
		}

	}

}

File: Web/Models/AuthLoginModel.cs
using Web.Models.Url;

namespace Web.Models{

	public class AuthLoginModel : PageModel {

	    public string ReturnUrl { get; set; }
	    public UserAddUrlModel AddUserUrl { get; set; }
	    public ForgotPasswordUrlModel ForgotPasswordUrl { get; set; }
	    public string LoginName { get; set; }
        public string Password { get; set; }
	    public bool RememberMe { get; set; }

        public AuthLoginModel() : base(null)
	    {
			
	    }

	}

}